packages/headless/src/controllers/facets/category-facet/headless-category-facet-options.ts

```diff
@@ -17,14 +19,31 @@ import {
   facetSearch,
 } from '../_common/facet-option-definitions';
 
-type OptionalFacetId = Partial<
-  Pick<CategoryFacetRegistrationOptions, 'facetId'>
->;
+export interface CategoryFacetOptions extends BaseFacetOptions {
+  /**
+   * Facet search options.
+   */
+  facetSearch?: CategoryFacetSearchOptions;
+  /**
+   * The base path shared by all values for the facet.
+   * @default []
+   */
+  basePath?: string[];
+  /**
+   * Whether to use basePath as a filter for the results.
+   * @default true
+   */
+  filterByBasePath?: boolean;
+  /**
+   * The sort criterion to apply to the returned facet values.
+   *
+   * @default "automatic"
+   */
+  sortCriteria?: CategoryFacetSortCriterion;
+}
 
-export type CategoryFacetOptions = OptionalFacetId &
-  Omit<CategoryFacetRegistrationOptions, 'facetId'> & {
-    facetSearch?: Partial<FacetSearchRequestOptions>;
-  };
+// eslint-disable-next-line @typescript-eslint/no-empty-interface
+export interface CategoryFacetSearchOptions extends BaseFacetSearchOptions {}
```

Comment:
Some interfaces, like this one, are empty and only extend another interface. They are there for the purpose of exposing a properly named interface developers can use in which we can add stuff later if necessary.

An alternative would be to export a `type` or re-export with an alias, but I don't know if those are supported by the documentation generation yet.